addons/website/static/src/systray_items/edit_website.scss

```
@@ -1,5 +1,5 @@
 .o_edit_website_container {
     &:not(.o_translatable) {
-        @include button-variant($o-brand-primary, $o-brand-primary);
+        margin: $o-navbar-padding-v * -1 0;
```

Can we do this instead? I find it more readable 
```suggestion
        margin: -$o-navbar-padding-v 0;
```

---

Fixed 👍